### ISSUE TITLE
Need criteria and process for adding facades to web.dev documentation

### ISSUE THREAD

**Issue by adamraine** [GoogleChrome/lighthouse#11583]
[web.dev draft](https://docs.google.com/document/d/1WCDboioqEuoBGLKQ0UDe9-ptg41lEPk9-93wzTWrZ6M/edit#heading=h.phl518fi3dj0)
Third party facades audit: https://github.com/GoogleChrome/lighthouse/pull/11290

Would be nice to have when the third party facades audit releases but not a blocking issue.

-----
Update: [additions process draft created](https://docs.google.com/document/d/1d1swpZNBgTdBdoNagU0KIjjOeUnSbxenv8pAlUbw7Pk/edit#heading=h.2o17r9iz1b9f)

**Comment by brendankenny** [comment]
> Would be nice to have when the third party facades audit releases but not a blocking issue.

yes, I was wondering about #11290 and if you wanted to wait to land the article first. Does that mean you'd rather land the article first and then the article?

**Comment by adamraine** [comment]
> Does that mean you'd rather land the audit first and then the article?

The web.dev article needs to be done before landing the audit.

This issue is for defining the process to add new facades to web.dev in the future. This needs to be done at some point but is not integral to the audit.

**Comment by adamraine** [comment]
Docs are being created in Third Party Web: https://github.com/patrickhulce/third-party-web/pull/119

**Comment by adamraine** [comment]
Docs are merged
